P9Operators/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿// mandel
+
+Console.WriteLine("Give me a number of seconds: ");
+int userInput = int.Parse(Console.ReadLine());
+
+/*
+int days = userInput / 86400;
+int remainingH = userInput % 86400;
+int hours = remainingH / 3600;
+int remainingM = remainingH % 3600;
+int min = remainingM / 60;
+int remainingS = remainingM % 60;
+*/
+
+int days = Math.DivRem(userInput, 86400, out int remainingH);
```

Comment:
Instead of these magic numbers, consider using variables:

```csharp
int minuteToSeconds = 60;
int hourToSeconds = 60*minuteToSeconds;
int dayToSeconds = 24*hourToSeconds;

doubleTotalDaysFraction = (double)totalSeconds / dayToSeconds;
```